tests/Field/WeekOfYearTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\DateTime\Tests\Field;
+
+use Brick\DateTime\Field\WeekOfYear;
+use Brick\DateTime\Tests\AbstractTestCase;
+
+/**
+ * Unit tests for class WeekOfYear.
+ */
+class WeekOfYearTest extends AbstractTestCase
+{
+    /**
+     * @expectedException        Brick\DateTime\DateTimeException
+     * @expectedExceptionMessage Invalid week-of-year: -1 is not in the range 1 to 53.
+     */
+    public function testCheckShouldThrowDateTimeExceptionWithFieldNotInRange()
+    {
+        WeekOfYear::check(-1);
+    }
+
+    /**
+     * @expectedException        Brick\DateTime\DateTimeException
+     * @expectedExceptionMessage Year 2000 does not have 53 weeks
+     */
+    public function testCheckShouldThrowDateTimeExceptionWithNonLeapYear()
```

Comment:
Sorry, my mistake, it's not a question of leap year here.
The name should be testCheckShouldThrowDateTimeExceptionWith52WeekYear()